constants/appLinks.ts

```diff
@@ -0,0 +1,170 @@
+type SegmentData = {
+  action: string
+}
+
+/**
+ * Represent a navigation link for the social media community links of the site
+ */
+interface SocialMediaCommunityLink {
+  /** The visible name of the link */
+  name: string
+  /** Where we want to go */
+  url: string
+  /** Includes an object with the information of the action. Example:
+   * ```ts
+   * {
+      action: 'Qiskit Community: GitHub'
+     }
+    ```
+   */
+  segment: SegmentData
+  /** Text shown when leaving the cursor on the link */
+  title: string,
+  /** Icon definition */
+  iconPath: string,
+  /** Use when a second path for the icon is needed */
+  iconPathSecondPart?: string
+}
+
+/**
+ * Represent a general link use on the site
+ */
+interface GeneralLink {
+  /** The visible name of the link */
+  label: string
+  /** Where we want to go */
+  url: string
+  /** Includes an object with the information of the action. Example:
+   * ```ts
+   * {
+      action: 'Qiskit Community: GitHub'
+     }
+    ```
+   */
+  segment?: SegmentData
+  /** Text shown when leaving the cursor on the link */
+  title?: string,
+}
+
+const SOCIAL_MEDIA_COMMUNITY: Array<SocialMediaCommunityLink> = [
+  {
+    name: 'Slack',
+    url: 'http://ibm.co/joinqiskitslack',
+    title: 'Slack community',
+    segment: {
+      action: 'Qiskit Community: Slack'
+    },
+    iconPath: 'M9.879 10.995l1.035 3.085 3.205-1.074-1.035-3.074-3.205 1.08v-.017z',
```

Comment:
Please, provide the complete SVG.

Splitting the SVG in their components has two main disadvantages:
1. It is non-trivial to re-compose the SVG to figure out what is drawn.
2. It is hard to edit, or accommodate to a new format (in case an icon needs three paths instead of two).

I'm also in favour of referring to an external SVG and embedding with a plugin, in the spirit of [`vue-inline-svg`](https://www.npmjs.com/package/vue-inline-svg) or something better.

Reply:
Per offline conversations, we will leave this point for the moment because svg icons probably change to carbon design icons